### PR TITLE
added styles

### PR DESCRIPTION
**Pull request by andrii-kuskovets**

### CODE REVIEW

**Comment by ladomiryak** [comment]
Remove eot, ttf fonts. use woff, woff2 only. it supports by all modern browsers.

File: app/icons/gmail.svg [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
+<g>
+</g>
```
Comment (ladomiryak) [inline app/icons/gmail.svg:41]: its useless. Use SVGOMG for SVG optimisation.

File: app/index.html [hunk 2]
```diff
@@ -4,179 +4,222 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Limitedadwear</title>
+    <link rel="stylesheet" href="css/style.min.css">
 </head>
 <body>
     <div class="out">
```
Comment (ladomiryak) [inline app/index.html:10]: what is out?

Comment (ladomiryak) [reply app/index.html:10]: add meaningful class names

File: app/index.html [hunk 3]
```diff
@@ -4,179 +4,222 @@
...
-            <input type="text" class="header__search" placeholder="Search">
-            <div class="header__cart">
-                <span>cart</span>
-                <span></span>
+            <div class="header__wrapper">
+                <div class="header__logo">
+                    <a href="#"><img src="img/logo.png" alt="" class="header__img"></a>
+                </div>
+                <div class="header__navigation">
+                    <ul class="header__menu">
```
Comment (ladomiryak) [inline app/index.html:28]: should be wrapped by `nav` tag.

File: app/index.html [hunk 4]
```diff
@@ -4,179 +4,222 @@
...
-                <li><a href="#">gift sertificates</a></li>
-            </ul>
-        </nav>
-        <div class="hamurger">
-            <span></span>
-            <span></span>
-            <span></span>
+                </ul>
+            </nav>
+            <div class="hamburger">
```
Comment (ladomiryak) [inline app/index.html:80]: its a `button` not  `div`

File: app/js/script.js [hunk 5]
```diff
@@ -2,38 +2,49 @@ $(document).ready(function(){
...
+        $('.promo_js').hide('slow');
     });
 
     //search
     $(".search__icon").on('click', function(){
-        $(".search, .search__input").toggleClass("active");
-        $(".search__input[type='text']").focus();
+        $(".search__form").toggleClass("search__form_active");
+        $(document).click( function(e){
+            if ( $(e.target).closest('.search').length ) {
```
Comment (ladomyriak) [inline app/js/script.js:12]: input .focusout() trigger the same action

File: app/js/script.js [hunk 6]
```diff
@@ -0,0 +1,126 @@
...
+        $(a).on('click', function () {
+            $(this).parent().siblings().find(b).hide();
+            $(this).parent().siblings().find('.menu__link_icon').removeClass('menu__link_icon_active');
+            $(this).siblings(b).slideToggle("slow");
+            $(this).toggleClass('menu__link_icon_active');
+        });
+    }
+
+    open($('.js_title'), $('.js_menu'));
+    open($('.js_subtitle'), $('.js_submenu'));
```
Comment (ladomiryak) [inline app/js/script.js:47]:
```
  function toggleMenu() {
    $(this).parent().siblings().find("ul").hide();
    $(this)
      .parent()
      .siblings()
      .find(".menu__link_icon")
      .removeClass("menu__link_icon_active");
    $(this).siblings("ul").slideToggle();
    $(this).toggleClass("menu__link_icon_active");
  }

  $(".js_title, .js_subtitle").click(toggleMenu);
```

better to use `$(selector).action(function...)`   it's more readable code.

File: app/js/script.js [hunk 7]
```diff
@@ -0,0 +1,126 @@
...
+        slidesToShow: 1,
+        slidesToScroll: 1,
+        arrows: false,
+        dots: true,
+        autoplay: true,
+        autoplaySpeed: 2000
+    });
+
+    //tabs
+    function slide(a, b) {
```
Comment (ladomiryak) [inline app/js/script.js:91]: why tabs contain `slide` function?

File: app/js/script.js [hunk 8]
```diff
@@ -0,0 +1,126 @@
...
+            $(this).next(a).slideToggle();
+            $(this).toggleClass('active');
+        });
+    }
+
+    slide($('.tabs__name'), $('.tabs__description'));
+    slide($('.accordion__title'), $('.accordion__list'));
+
+    //changes size&color btns
+    function changes(a, b) {
```
Comment (ladomiryak) [inline app/js/script.js:104]: same here. changes for what? name it properly, please.

File: app/scss/blocks/_product.scss [hunk 9]
```diff
@@ -26,14 +26,47 @@
...
         color: $grey;
     }
 
+    &__buttons {
+        display: flex;
+        flex-direction: column;
+        align-items: flex-start;
+    }
+
+    //icons for btn
```
Comment (ladomiryak) [inline app/scss/blocks/_product.scss:48]: is it needed? comment

File: dist/js/script.js [hunk 10]
```diff
@@ -2,38 +2,49 @@ $(document).ready(function(){
 
     //close promo text
     $('.promo__close').on('click', function () {
-        $('.promo_hide').hide('slow');
+        $('.promo_js').hide('slow');
```
Comment (ladomiryak) [inline dist/js/script.js:5]: rename class to `js_promo` as you have it elsewhere

Comment (ladomiryak) [reply dist/js/script.js:5]: but now you have a mix of classes. if you want to use `js` prefixes use it everywhere. so you will have:
`js-promo-close` class for calling action...

File: dist/js/script.js [hunk 11]
```diff
@@ -2,38 +2,49 @@ $(document).ready(function(){
...
     });
 
     //search
     $(".search__icon").on('click', function(){
```
Comment (ladomiryak) [inline dist/js/script.js:9]: `js-search` for example here..

File: dist/js/script.js [hunk 12]
```diff
@@ -2,38 +2,49 @@ $(document).ready(function(){
...
+            if ( $(e.target).closest('.search').length ) {
+                return;
+            }
+            $('.search__form').removeClass("search__form_active");
+        });
     });
 
     //menu
     
     $('.hamburger').on('click', function () {
```
Comment (ladomiryak) [inline dist/js/script.js:21]: `js-mobile-menu`

File: app/scss/blocks/_promo.scss [hunk 13]
```diff
@@ -0,0 +1,46 @@
...
+            color: $white;
+        }
+    }
+    
+    &__phrase {
+        color: $blue;
+        text-transform: uppercase;
+
+        &_bold {
+            font-family: 'Proxima-semibold', serif;
```
Comment (BearSwan) [inline app/scss/blocks/_promo.scss:28]: add variable

**Pull request merged**